src/main/java/com/dumbdogdiner/stickyapi/common/util/Debugger.java

```diff
@@ -51,18 +51,18 @@
     @Getter
     private int logCount = 0;
 
-    private Class<?> clazz;
+    private final Class<?> debuggedClass;
```

Comment:
`clazz` is actually conventional for declaring a class variable, but this is fine and i'm not requesting a change.

Reply:
Ah, I was not aware of this. I won't change it in the future.